coriolis/providers/replicator.py

```diff
@@ -38,6 +36,9 @@
     cfg.IntOpt('port',
                default=DEFAULT_REPLICATOR_PORT,
                help='The replicator TCP port.'),
+    cfg.IntOpt('replicator.default_requests_timeout',
```

Comment:
The extra `replicator.` prefix is not needed when defining the config options, the group name is passed as a second argument to `CONF.register_opts` below.